File: programme/area-of-trapezoid/area-of-trapezoid.cs
// Write a  program to compute area of trapezoid
using System;

class MainClass
{
    public static void Main(string[] args)
    {
        Console.Write("Base 1 : ");
        double base1 = Convert.ToDouble(Console.ReadLine());

        Console.Write("Base 2 : ");
        double base2 = Convert.ToDouble(Console.ReadLine());

        Console.Write("Height : ");
        double height = Convert.ToDouble(Console.ReadLine());

        double area = (base1 + base2) * height / 2;
        Console.WriteLine("\nThe area of trapezoid is : " + area);

    }
}

File: programme/area-of-trapezoid/area-of-trapezoid.jl
# Write a programme to compute area of trapezoid

using Printf

print("Base 1 : ")
b1 = parse(Int32, readline())

print("Base 2 : ")
b2 = parse(Int32, readline())

print("Height : ")
h = parse(Int32, readline())


@printf("\nArea : %0.2f", (b1 + b2) * h * 0.5)
